ui/src/views/IpBlacklist.vue

```diff
@@ -256,11 +256,14 @@
                   :class="['checkbox-label', {'gray': (!props.row.enabled || !config.status)}]"
                 >
                   <span
-                    :class="['category-status-icon', 'pficon', (props.row.enabled && config.status) ? ['pficon-ok', 'green'] : 'pficon-off']"
+                    :class="['category-status-icon', 'pficon', (props.row.missing && config.status) ? ['pficon-warning-triangle-o', 'orange'] : (props.row.enabled && config.status) ? ['pficon-ok', 'green'] : 'pficon-off']"
                   ></span>
-                  <span
+                  <span v-if="(!props.row.missing && config.status)"
                     :class="{'green': (props.row.enabled && config.status)}"
                   >{{ (props.row.enabled && config.status) ? $t('enabled') : $t('disabled') }}</span>
+                  <span v-if="(props.row.missing && config.status)"
```

Comment:
See comment in `DnsBlacklist.vue`

Reply:
We have to check the config.status if false all categories are disabled (grayed)